src/openlifu/db/database.py

```diff
@@ -192,7 +192,7 @@ def write_subject(self, subject, on_conflict=OnConflictOpts.ERROR):
 
         self.logger.info(f"Added subject with ID {subject_id} to the database.")
 
-    def write_transducer(self, transducer, on_conflict: OnConflictOpts=OnConflictOpts.ERROR):
+    def write_transducer(self, transducer, registration_surface_model_filepath: Optional[str] = None, transducer_body_model_filepath: Optional[str] = None, on_conflict: OnConflictOpts=OnConflictOpts.ERROR):
```

Comment:
this is a good time to add a docstring to this function explaining for example what is meant by registration surface and body model